modules/cdn_with_s3_bucket/bucket.tf

```diff
@@ -25,11 +25,16 @@ data "aws_iam_policy_document" "s3_origin" {
 }
 
 resource "aws_s3_bucket" "default" {
-  #checkov:skip=CKV_AWS_21:Versioning is disabled to save money
-  #checkov:skip=CKV_AWS_144:Cross-region replication is disabled to save money
   #checkov:skip=CKV_AWS_18:Access logging is disabled to save money
   #checkov:skip=CKV_AWS_19:Data are encrypted - outdated rule
+  #checkov:skip=CKV_AWS_21:Versioning is disabled to save money
+  #checkov:skip=CKV_AWS_144:Cross-region replication is disabled to save money
   #checkov:skip=CKV_AWS_145:Data are encrypted - outdated rule
+  #checkov:skip=CKV2_AWS_37:Ensure all data stored in the S3 bucket have versioning enabled
+  #checkov:skip=CKV2_AWS_38:S3 Bucket has an ACL defined which allows public READ access.
+  #checkov:skip=CKV2_AWS_39:S3 Bucket has an ACL defined which allows public WRITE access.
+  #checkov:skip=CKV2_AWS_40:Ensure all data stored in the S3 bucket is securely encrypted at rest
+  #checkov:skip=CKV2_AWS_41:Ensure all data stored in the S3 bucket have logging
```

Comment:
Shouldn't those comments explain why those checks are skipped?

Reply:
You are right - my fault